### DISCUSSION TITLE
Release date for Data Blocks?

### DISCUSSION THREAD

**Discussion by gvlx** [Budibase/budibase#3647] [Help]
The functionality described here https://github.com/Budibase/budibase/pull/3355 does not seem be available in the present version (1.0.3).

Is there a planned date for this?

Array fields as a source of data seem very powerful and interesting.

**Comment by shogunpurple** [comment]
Hi @gvlx - this functionality is now available in both cloud and self-host. We generally perform the release without comms, and send out the communications in discussions/social a few days later.
